src/main/java/core/mate/academy/service/TruckProducer.java

```diff
@@ -0,0 +1,21 @@
+package core.mate.academy.service;
+
+import core.mate.academy.model.Truck;
+import java.util.List;
+
+public class TruckProducer implements MachineProducer<Truck> {
```

Comment:
The class declaration should not be followed by an empty line. Remove the empty line after the class declaration to adhere to best practices.